proto/nexus/v1beta1/service.proto

```diff
@@ -69,12 +69,12 @@ service QueryService {
 
   // Fee queries the fee info by chain and asset
   rpc Fee(FeeRequest) returns (FeeResponse) {
-    option (google.api.http).get = "/nexus/v1beta1/fee";
+    option (google.api.http).get = "/axelar/nexus/v1beta1/fee";
```

Comment:
Why prefix the url with `/axelar`? I don't see this convention in other modules.

Reply:
Does it hurt to fix those that are not prefixed in the PR?

Reply:
I'm not sure if MS or others depend on these REST endpoints somehow. Probably not, but keeping that as a separate issue #1379